manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -27,6 +27,8 @@ spec:
         sidecar.istio.io/inject: "false"
 {{ with .Values.podAnnotations -}}{{ toYaml . | indent 8 }}{{ end }}
     spec:
+      nodeSelector:
```

Comment:
better to make this configrable?

Reply:
Are there scenarios where you'd want to exclude certain nodes from Ambient besides non-linux nodes?

Reply:
not a strong option